diff --git a/apps/hierarchy/src/src/libs/firestore.js b/apps/hierarchy/src/src/libs/firestore.js
index d114fbb..09ea0bf 100644
--- a/apps/hierarchy/src/src/libs/firestore.js
+++ b/apps/hierarchy/src/src/libs/firestore.js
@@ -33,6 +33,7 @@ function getStorage() {
 
     if (process.env.FIRESTORE_EMULATOR_HOST) {
         settings.host = config.FIRESTORE_EMULATOR_HOST;
+        settings.projectId = config.FIRESTORE_PROJECT_ID;
     } else {
         settings.databaseId = config.FIRESTORE_DATABASE_ID;
         settings.projectId = config.FIRESTORE_PROJECT_ID;
