diff --git a/apps/portal/src/backend/mappings/tenant-tags.json b/apps/portal/src/backend/mappings/tenant-tags.json
new file mode 100644
index 000000000..6269b6b01
--- /dev/null
+++ b/apps/portal/src/backend/mappings/tenant-tags.json
@@ -0,0 +1,40 @@
+{
+  "aliases": {},
+  "mappings": {
+    "dynamic_templates": [
+      {
+        "dynamic_objects_as_keywords": {
+          "match": "*",
+          "mapping": {
+            "type": "keyword",
+            "normalizer": "lowercase"
+          }
+        }
+      }
+    ],
+    "dynamic": "strict",
+    "properties": {
+      "tenantId": {
+        "type": "keyword",
+        "normalizer": "lowercase"
+      },
+      "tag": {
+        "type": "keyword",
+        "normalizer": "lowercase",
+        "fields": {
+          "search": {
+            "type": "text"
+          }
+        }
+      },
+      "entity": {
+        "type": "keyword",
+        "normalizer": "lowercase"
+      },
+      "last_update_date": {
+        "type": "date",
+        "format": "epoch_millis"
+      }
+    }
+  }
+}
diff --git a/apps/portal/src/backend/script/migrations/createRootHierarchies.js b/apps/portal/src/backend/script/migrations/createRootHierarchies.js
index 399526432..2853c0b20 100644
--- a/apps/portal/src/backend/script/migrations/createRootHierarchies.js
+++ b/apps/portal/src/backend/script/migrations/createRootHierarchies.js
@@ -31,9 +31,9 @@ const rootOrganizationName = 'Root';
  */
 const run = async () => {
     const token = await getAuthToken();
-    const tenantIds = await getTenantIds(token);
-
-    for (const tenantId of tenantIds) {
+    // const tenantIds = await getTenantIds(token);
+    const tenantId = "1";
+    // for (const tenantId of tenantIds) {
         try {
             await new HierarchyConfigActivateService().activate({
                 rootOrganizationId,
@@ -56,7 +56,7 @@ const run = async () => {
 
             log.error(err, `An unexpected error occurred for tenant "${tenantId}": ${err.message}`);
         }
-    }
+    // }
 };
 
 run();
diff --git a/apps/portal/src/backend/src/controllers/lti.controller.js b/apps/portal/src/backend/src/controllers/lti.controller.js
index 6b8910906..c426fa53a 100644
--- a/apps/portal/src/backend/src/controllers/lti.controller.js
+++ b/apps/portal/src/backend/src/controllers/lti.controller.js
@@ -223,7 +223,7 @@ async function launchAuthoring(req, res, next) {
 
         const clientId = await sidecarGateway.getAuthoringClientId(tenantId);
         const ltiRegistration = await fetchRegistration(tenantId, clientId);
-
+        log.debug(JSON.stringify(ltiRegistration));
         const tenantClaims = await sidecarGateway.getTenantLtiClaimsForDeliveryLaunch(tenantId);
         log.debug(`tenantClaims: ${JSON.stringify(tenantClaims)}`);
         const locale = await sidecarGateway.getTenantLocale(tenantId);
@@ -234,7 +234,8 @@ async function launchAuthoring(req, res, next) {
         }
 
         ltiLaunchLink = await ltiUtil.generateLtiLaunchLink(ltiRegistration, {
-            targetLinkUrl: '/taoLti/AuthoringTool/launch',
+            // targetLinkUrl: '/backoffice/taoLti/AuthoringTool/launch',
+            ltiLaunchUrl: ltiRegistration.tool.launchUrl,
             user: req.token.user,
             claims: {
                 ...tenantClaims,
diff --git a/apps/portal/src/backend/src/services/gateway/tag.gateway.js b/apps/portal/src/backend/src/services/gateway/tag.gateway.js
index 5061a6cd3..3e76cfe31 100644
--- a/apps/portal/src/backend/src/services/gateway/tag.gateway.js
+++ b/apps/portal/src/backend/src/services/gateway/tag.gateway.js
@@ -28,7 +28,8 @@ const { log } = require('../logger.service');
  * @throws {Error} will throw connection and application errors
  */
 async function applyTags(entityName, id, tags) {
-    if (!tags || !Array.isArray(tags)) return Promise.resolve();
+    return Promise.resolve();
+    // if (!tags || !Array.isArray(tags)) return Promise.resolve();
     try {
         log.trace(entityName, id, tags);
         await axios.post(`${config.classification.apiUrl}/api/v1/tags/apply`, {
diff --git a/apps/portal/src/backend/src/services/gateway/tenantApi.gateway.js b/apps/portal/src/backend/src/services/gateway/tenantApi.gateway.js
index b1828b743..e07847a27 100644
--- a/apps/portal/src/backend/src/services/gateway/tenantApi.gateway.js
+++ b/apps/portal/src/backend/src/services/gateway/tenantApi.gateway.js
@@ -134,13 +170,8 @@ function getHeaders(token) {
 
 async function getTenantIds(token) {
     try {
-        const url = `${config.tenantApi.apiUrl}/api/v1/tenants`;
-
-        log.debug(`Getting tenant ids`);
-
-        const response = await axios.get(url, getHeaders(token));
+        return ["1"];
 
-        return response.data.map(tenantData => tenantData.meta.tenantId);
     } catch (err) {
         log.error(err, `Error getting tenant ids: error=%s`, err.message);
         throw new PortalError(500, `Error getting tenant ids: ${err.message}`);
